app/assets/javascripts/models/page/project_page.js.coffee

```diff
@@ -88,6 +88,29 @@ ns.ProjectPage = do (ko,
     hasType: =>
       @query.serialize().bounding_box || @query.serialize().polygon || @query.serialize().point
 
+    showTemporal: =>
+      label = ""
+      if @query.serialize().temporal
+        dates = @query.serialize().temporal.split(",")
+        
+        dateStart = moment(dates[0].split("T")[0]) 
```

Comment:
What was determined here regarding splitting out the date on the 'T'? Moment is capable of parsing the date without stripping information. I'm afraid to do this type of thing because it's specific to the date format currently being used, but it doesn't have to be.

Reply:
@abbottry - I wasn't able to split it all out.  If that split 'T' doesn't happen, it has a warped date (months or years off).  The other splits were 'okay' to remove.

Reply:
I'm not able to reproduce that http://jsfiddle.net/remus/rLjQx/ Seems to work fine..

Reply:
I invite you to try it within the application - or, I could try it and post screenshots of it not working.  :\

Reply:
@abbottry - gah, let me toy with it just a bit more.  At a minimum I'll  see if I can find an explanation.

Reply:
Working for me. Moment takes into account the Timezone, so maybe you were seeing the timezone adjustment that moment does on display but moment has a very very good parser and this is a standard date format so I'd expect it to handle it just fine. 

![screen shot 2017-12-06 at 2 00 49 pm](https://user-images.githubusercontent.com/174744/33679858-e475df2e-da8d-11e7-9ef5-4dd10aef44a5.png)